src/components/victory-chart/victory-chart.jsx

```diff
@@ -129,27 +124,10 @@ export default class VictoryChart extends React.Component {
   };
 
   componentWillReceiveProps(nextProps) {
-    if (!this.props.animate) {
-      return;
-    }
-
-    const {
-      nodesWillExit,
-      nodesWillEnter,
-      childrenTransitions,
-      nodesShouldEnter
-    } = Transitions.getInitialTransitionState(this.props.children, nextProps.children);
-
-    this.setState({
-      nodesWillExit,
-      nodesWillEnter,
-      childrenTransitions,
-      nodesShouldEnter,
-      oldProps: nodesWillExit ? this.props : null
-    });
+    const setAnimationState = Wrapper.setAnimationState.bind(this);
+    setAnimationState(nextProps);
```

Comment:
Since you're immediately invoking the bound method here, I would suggest either
1) Pass the component instance `this` into the helper method as a param named `component`, or
2) `Wrapper.setAnimationState.call(this, nextProps)`